src/main/kotlin/com/hibob/project/dao/employeeDao.kt

```diff
@@ -0,0 +1,38 @@
+package com.hibob.project.dao
+
+import jakarta.ws.rs.NotFoundException
+import org.jooq.DSLContext
+import org.jooq.RecordMapper
+import org.jooq.Record
+import org.springframework.stereotype.Component
+
+@Component
+class EmployeeDao(private val sql: DSLContext) {
+    private val employeeTable = EmployeesTable.instance
+    private val employeeMapper = RecordMapper<Record, Employee>
+    { record ->
```

Comment:
it's better if it is in the assignment line, so it's clear that it belongs there